_drafts/semantic-release.md

```diff
@@ -0,0 +1,37 @@
+---
+state: In Progress
+---
+
+# Set up release workflow with `semantic-release`
+
+`semantic-release` makes releasing faster, simpler and safer. All of these are valuable if we want to increase public participation in the project and increase momentum and motivation.
+
+## Motivation
+
+Releasing a package is always a bit of a Big Thing™: decisions have to be made about versioning, an error-prone manual release process has to be performed, and somebody has to actually do the work and take the responsibility if something goes wrong. So there are a few barriers to frequent releases, and  many potential sources of error. Automating this process takes away all the uncertainty and hassle out of releasing, makes our version numbers more consistent and reliable, and increases release frequency and project momentum. It also does away with the concept of intentional, manual releases: whenever something lands in master and passes all the tests, `semantic-release` will package an npm release. If we do need to keep control of release timing for marketing reasons, `semantic-release` supports that too.
+
+## Proposed solution
+
+Set up `semantic-release` on `hubotio/hubot` and let it release the npm module for us automatically and continuously.
+
+## Detailed design
+
+An admin/owner [sets up](https://github.com/semantic-release/semantic-release#setup) [semantic-release](https://github.com/semantic-release/semantic-release) on `hubotio/hubot` using `semantic-release-cli setup`. Since we’ve already got tests and Travis set up, this shouldn’t require any additional steps.
+
+We should update `CONTRIBUTING.md`:
+- include more details about our [expected commit message format](https://github.com/semantic-release/semantic-release#default-commit-message-format), and why this is important
```

Comment:
I would suggest we put the most common commit messages into our `CONTRIBUTING.md` files. `fix: ...`, `feat: ...` and `BREAKING CHANGE:` are the only three that really matter for `semantic-release` as only they have impact on the calculated version and only these commit messages will be parsed for changes added to a GitHub release.

We have a short summary in [Hoodie’s CONTRIBUTING.md file](https://github.com/hoodiehq/hoodie/blob/master/CONTRIBUTING.md#commit-message-conventions)